samples/charts/line/stepped.html

```diff
@@ -29,14 +29,14 @@
 <body>
 	<div class="container">
 	</div>
-    <script>
+		<script>
 	function createConfig(details, data) {
 		return {
 			type: 'line',
 			data: {
 				labels: ['Day 1', 'Day 2', 'Day 3', 'Day 4', 'Day 5', 'Day 6'],
 				datasets: [{
-					label: 'steppedLine: ' + ((typeof(details.steppedLine) === 'boolean') ? details.steppedLine : `'${details.steppedLine}'`),
+					label: 'steppedLine: ' + ((typeof (details.steppedLine) === 'boolean') ? details.steppedLine : details.steppedLine), // TODO : check this
```

Comment:
not sure what's going on here

Reply:
`typeof` as operator without parenthesis?

Reply:
no, i mean why we have 
``` 
? details.steppedLine : `'${details.steppedLine}'`

Reply:
I think `typeof(foo)` is invalid because it's an operator so should not be used with parenthesis. We should write `typeof foo`, not `typeof (foo)` (with the extra space).

Anyway, you right, this line is weird, could be: `label: 'steppedLine: ' + details.steppedLine`